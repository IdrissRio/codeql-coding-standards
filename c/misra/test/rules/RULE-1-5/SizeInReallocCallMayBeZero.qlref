rules/RULE-1-5/SizeInReallocCallMayBeZero.ql