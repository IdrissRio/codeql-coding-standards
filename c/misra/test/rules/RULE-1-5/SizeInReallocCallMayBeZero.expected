| test.c:15:3:15:9 | call to realloc | Size argument '$@' equals zero in realloc call, resulting in obsolescent and/or implementation-defined behavior. | test.c:15:14:15:15 | p0 | p0 |
